src/dmd/parse.d

```diff
@@ -4411,6 +4411,24 @@ final class Parser(AST) : Lexer
         if (!comment)
             comment = token.blockComment.ptr;
 
+        /* Look for AliasAssignment:
+         *   identifier = type;
+         */
+        if (token.value == TOK.identifier && peekNext() == TOK.assign)
+        {
+            const loc = token.loc;
+            auto ident = token.ident;
+            nextToken();
+            nextToken();        // advance past =
```

Comment:
make this a check(TOK.equals) at the very least, please.

Reply:
Not necessary given line 4417

Reply:
Just because its not necessary doesn't mean that you shouldn't do it.